styleguide/components/21-atoms/inputs/input-multiselect-tagging/input-multiselect-tagging.bindings.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * Javascript binding of baguettebox.min.js.
+ */
+'use strict';
+
+(function () {
+
+  var elements = document.getElementsByClassName('multiselect-tagging'); // eslint-disable-line no-undef
+
+  for (var i = 0; i < elements.length; i++) {
```

Comment:
var i = elements.length; i--;

(in future: let)

Reply:
Fixed.